go.mod

```diff
@@ -188,6 +188,8 @@ require (
 	cloud.google.com/go/redis v1.10.0
 	github.com/Azure/azure-sdk-for-go/sdk/storage/azblob v0.4.1
 	github.com/Azure/go-autorest/autorest/adal v0.9.14
+	github.com/apache/arrow/go/arrow v0.0.0-20211112161151-bc219186db40
```

Comment:
There must have an incorrect import somewhere. We should only have a single version of the library.